libs/dynamixel/dynamixel.ts

```diff
@@ -0,0 +1,214 @@
+namespace network {
```

Comment:
Is this the namespace you want?

Reply:
There is no clear design for the block category yet. (Because this is not my authority ...)

Therefore, I have created based on existing categories. It may change in the future. Of course, blocks can also be changed.

Do you want us to PR after deciding the final block(include name, category)?

Reply:
Should Namespace be lowercase?
Do you have any documentation on pxt's coding style?

Reply:
You can merge and fix later. https://makecode.com/extensions/naming-conventions

Reply:
Done on 86f708f.